df_py/util/dftool_module.py

```diff
@@ -803,6 +803,47 @@ def do_new_acct():
     print(f" For other dftools: export DFTOOL_KEY={account.private_key}")
 
 
+# ========================================================================
+def do_dummy_csvs():
+    parser = argparse.ArgumentParser(description="Generate dummy CSVs")
+    parser.add_argument("command", choices=["dummy_csvs"])
+    parser.add_argument("SUBSTREAM", choices=["volume", "challenge", "predictoor"])
+    parser.add_argument(
+        "CSV_DIR", type=autocreate_path, help="output dir for csv files"
+    )
+    arguments = parser.parse_args()
+    print_arguments(arguments)
+
+    CSV_DIR, SUBSTREAM = arguments.CSV_DIR, arguments.SUBSTREAM
```

Comment:
I think we should change the capitalisation on these variables, as well. I thought I did, as part of the capitalisation run, but apparently I didn't. We can create a separate ticket to have all these clearly capitalised as variables, not constants.